EventListener/ReplaceImageListener.php

```diff
@@ -0,0 +1,126 @@
+            sprintf('<img alt="Exception detected!" data-gif src="%s" style="width:145px" />', $this->generateUrl(sprintf('bundles/gifexception/images/%s/%s', $dir, $gif))),
```

Comment:
Maybe you should use a preg_replace here instead, because the actual Symfony ghost could change one day.